packages/vite/src/node/config.ts

```diff
@@ -571,11 +563,14 @@ export async function resolveConfig(
 
   const optimizeDeps = config.optimizeDeps || {}
 
-  if (process.env.VITE_TEST_LEGACY_CJS_PLUGIN) {
-    config.legacy = {
-      ...config.legacy,
-      buildRollupPluginCommonjs: true
-    }
+  if (process.env.VITE_TEST_DISABLING_CJS_PLUGIN) {
```

Comment:
Should we update this env at 

https://github.com/vitejs/vite/blob/2185f7236e91361304b197de8c9b84b66511be73/package.json#L24

too?

Reply:
Done 👍🏼 
Changed the name to test-build-without-plugin-commonjs